provider/signal_rule_resource_test.go

```diff
@@ -40,6 +40,7 @@ func testAccFireHydrantSignalRuleConfigBasic() string {
 		expression = "signal.summary == 'test-signal-summary'"
 		target_type = "User"
 		target_id = data.firehydrant_user.test_user.id
+		incident_type_id = "3314f092-8632-4412-aba8-677ba88d035b"
```

Comment:
I probably would duplicate this test to make sure that when `incident_type_id` is missing, we don't panic by accident or terraform constantly thinking we need to update, since we're doing a lot of `.(string)` casting.